src/ildasm/windasm.cpp

```diff
@@ -383,6 +383,9 @@ int ProcessOneArg(__in __nullterminated char* szArg, __out char** ppszObjFileNam
         }
         else if ((_stricmp(szOpt, "met") == 0)&&g_fTDC)
         {
+#ifdef FEATURE_CORECLR
+            printf("Warning: 'METADATA' option is ignored for ildasm on CoreCLR.\n");
+#else
             if(g_fTDC)
```

Comment:
A little orthogonal to what you are fixing but happened to notice.

The if check on line 389 is redundant, since the if check on line 384 means that g_fTDC is non-null already.